.changeset/pre.json

```diff
@@ -0,0 +1,70 @@
+{
```

Comment:
Should we keep this change out of the PR since I guess we don't want pre-versioning happening on a stable release?